packages/frontend/src/hooks/useEcharts.ts

```diff
@@ -199,6 +210,9 @@ export const getEchartsSeries = (
                 ...sum,
                 {
                     ...series,
+                    emphasis: {
+                        focus: 'series',
```

Comment:
Do we also want series focus when not pivoting ?

Reply:
It works for pivoted and unpivoted for me:

<img width="660" alt="Screenshot 2022-03-28 at 17 01 09" src="https://user-images.githubusercontent.com/11660098/160439488-6986270c-fbe6-45c4-b360-42a37bbde60a.png">